pom.xml

```diff
@@ -87,6 +87,16 @@
     <dependencyManagement>
         <dependencies>
             <!-- overrides of imports -->
+            <dependency>
+                <groupId>com.powsybl</groupId>
+                <artifactId>powsybl-time-series-api</artifactId>
+                <version>6.6.1</version>
```

Comment:
define the version in the properties and add a comment for the override. See https://github.com/powsybl/powsybl-network-store-server/commit/50f73ab289be39777b03ed2b5d6adf3d8e2f80e2